src/heap/sharedMem.ml

```diff
@@ -860,6 +858,10 @@ module NewAPI = struct
 
   type size = int
 
+  let bsize_wsize bsize = bsize * Sys.word_size / 8
+
+  let addr_offset addr size = addr + bsize_wsize size
```

Comment:
`bsize_wsize` seems only used once, shall we inline it to avoid a new name to make people to understand the code easier?

Reply:
Unless you feel strongly, I think I prefer giving a name to the wsize->bsize conversion. You're right that it's used only once, but my subjective opinion is that a separate function is easier to read/understand.